YOLayout/YOLayout_IMP.h

```diff
@@ -124,25 +124,27 @@ typedef enum {
 - (CGRect)setFrame:(CGRect)frame view:(id)view options:(YOLayoutOptions)options;
 
 /*!
- Set the (sub)view frame.
- If we are calculating sizeThatFits, this doesn't actually set the views frame.
- Use this value instead of view.frame since the views frame might not have been set.
+ Center view with size in frame.
  
- @param frame Frame
- @param inRect Rect to optionally position in for YOLayoutOptionsCenter, YOLayoutOptionsCenterVertical, YOLayoutOptionsRightAlign, etc.
+ @param size Desired size
```

Comment:
Maybe something here suggesting that this isn't necessarily the size of the view when sizing? Maybe:

    @param size Desired size (or size hint if using YOLayoutOptionsSizeToFit)